CHANGELOG.asciidoc

```diff
@@ -26,6 +26,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 TinkerPop 3.3.0 (Release Date: NOT OFFICIALLY RELEASED YET)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Updated Docker build scripts to include Python dependencies (NOTE: users should remove any previously generated TinkerPop Docker images).
```

Comment:
so far, upgrade docs have really just been for "users" and "providers" not for "TinkerPop Developers" so i'm not sure where something like that would go.  It doesn't fit too well there, but I suppose you could add it to the "users" section and it would make sense. I suppose it would just need to be made clear that the "docker stuff" is for purpose of "building tinkerpop source" or something like that. I doubt many end users really spend much time building tinkerpop, but I could be wrong. 

Anyway, i don't feel strongly about it one way or the other - just my two cents.